libs/perun/models/src/lib/ConfigProperties.ts

```diff
@@ -128,6 +128,7 @@ export interface PerunConfig {
   // Optional
   auto_auth_redirect?: boolean;
   supported_languages?: string[];
+  enforce_consents?: boolean;
```

Comment:
Please, also add this property to our config documentation here: https://perunaai.atlassian.net/wiki/spaces/STRIBOG/pages/3440714/Configuration+of+new+GUI+applications

Reply:
done